lib/gnip/ruler/ruler.rb

```diff
@@ -181,12 +197,28 @@ def location_format_gnip
       end
     end
 
+    # Generate places string for the Gnip rules.
+    #
+    def places_format_gnip
+      place_string = ''
+      unless @places.nil?
+        @places.each do |place|
+          place_string = "place:#{place}"
+          place_string.prepend(" ") if @hashtags.any?
+        end
+        place_string
+      else
+        ''
+      end
+    end
+
     # Set Gnip rule variables to nil
     def set_rule_vars
```

Comment:
Unrelated to code changes in this commit, but it feels like these should be moved to `#initialize` since they are setting default values